src/raft/follower.go

```diff
@@ -159,7 +159,9 @@ func (r *Follower) processHeartbeatRequest(req *model.RaftRPCRequest) *model.Raf
 			}
 
 			// before change master need check gtid, if local gtid bigger than remote gtid degrade to INVALID
-			r.degradeToInvalid(&gtid, &req.GTID)
+			if r.getMembers() > 2 {
```

Comment:
Is that normal when a two-node MySQL cluster's Leader exists local commit?

Reply:
Is it to avoid misjudgments that could lead to an abnormal MySQL cluster?

Reply:
If the total number of nodes is 2, semi-synchronous replication may degrade to asynchronous replication, in some cases, high availability is affected.

Reply:
> If the total number of nodes is 2, semi-synchronous replication may degrade to asynchronous replication, in some cases, high availability is affected.

Ok, I know. But only to a certain extent, and some of the risks. Is it right?

Reply:
yes